lib/ryan.rb

```diff
@@ -18,9 +18,25 @@ class Ryan
   def_delegators :const,
     :name, :funcs, :type, :initialization_args, :func_by_name, :class?, :module?
 
+  def self.root
+    current_file_name = __FILE__
+    path_to_current_file = File.expand_path(current_file_name)
+    root_path = Pathname.new(path_to_current_file) + "../.."
+
+    return root_path
+  end
+
   # @param [Pathname, String] file
-  def initialize(file)
-    @sexp = RubyParser.new.parse File.read(file)
+  def initialize(file, mode=:file)
```

Comment:
I'd prefer we kept the same interface, but make it more flexible. How bout we delegate to `File`?
```ruby
input = File.read(input) if File.file?(input)
@sexp = RubyParser.new.parse input
```